lms/envs/common.py

```diff
@@ -3314,6 +3314,7 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     # not be used anywhere else.
     'DEFAULT_AUTHENTICATION_CLASSES': [
         'openedx.core.djangolib.default_auth_classes.DefaultSessionAuthentication',
+        'openedx.core.djangolib.default_auth_classes.DefaultJwtAuthentication',
```

Comment:
@feanil: I imagined `DefaultJwtAuthentication` would be added first. Unfortunately, it does make a difference.
1. We've typically added `JwtAuthentication` first in many places like the [ecommerce defaults](https://github.com/openedx/ecommerce/blob/b5a96f17f24e40d7e74016596944663c49d3b9ab/ecommerce/settings/base.py#L517), or a variety of [edx-platform overrides you can see here](https://github.com/search?q=repo%3Aopenedx%2Fedx-platform%20authentication_classes&type=code).
2. `JwtAuthentication` does not check `is_active`, so email validation does not affect it. However, `SessionAuthentication` does. These work differently, and order changes in what way, which really stinks. See https://github.com/openedx/public-engineering/issues/165 for details.
3. `JwtAuthentication` may also update the database based on contents. Since the LMS creates these JWTs, this shouldn't have any affect at this time. But it could, when and if another service started creating the JWTs.

Are you ok with putting `JwtAuthentication` first as we had it?

I'll leave it up to you if you want to add code comments that explain these differences in order.

Reply:
Yea, that makes sense.  I'll update and take some of these comments and put them in next to the list.